terraform/alarms.tf

```diff
@@ -23,7 +23,7 @@ resource "aws_cloudwatch_metric_alarm" "response_time_24h" {
   extended_statistic  = "p95"
   metric_name         = "TargetResponseTime"
   comparison_operator = "GreaterThanThreshold"
-  threshold           = 3
+  threshold           = 5
   period              = 86400
   datapoints_to_alarm = 1
```

Comment:
Might be worth tweaking this as well, as we know there are occasional slow requests, but we want an alarm. if there is repeated occurance.